eva-accession-core/src/main/java/uk/ac/ebi/eva/accession/core/model/eva/ClusteredVariantEntity.java

```diff
@@ -112,6 +125,11 @@ public Boolean isValidated() {
         return validated == null ? DEFAULT_VALIDATED : validated;
     }
 
+    @Override
+    public Integer getMapWeight() {
+        return mapWeight;
```

Comment:
agreed to put the same behaviour as in the validated flag. also agreed to add a test in eva-accession-core to check that the default value (mapWeight == 1) is not written in mongo, but it's ok when retrieving.